packages/components/.stylelintrc.js

```diff
@@ -0,0 +1,151 @@
+'use strict';
+
+module.exports = {
+  extends: [
+    'stylelint-config-standard-scss',
+    // 'stylelint-prettier/recommended',
+    // 'stylelint-config-prettier-scss',
+  ],
+  rules: {
+    // =============================
+    // PRETTIER OVERRIDES:
+    // =============================
+    // 'indentation': null, // to test with prettier
+    // "indentation": 2,
+    // "string-quotes": "double",
+
+    // =============================
+    // TO FIX:
```

Comment:
@alex-ju @KristinLBradley @MelSumner @Dhaulagiri this block of rules are the ones that have been already applied in https://github.com/hashicorp/design-system/pull/545 - once that PR is merged we can consider these approved (some declarations will be removed because are default).